obspy/io/hypodd/tests/test_pha.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+
+import os
+import unittest
+
+from obspy import read_events, read_inventory
+from obspy.io.hypodd import pha
+
+
+class PHATestCase(unittest.TestCase):
+    """
+    Test suite for obspy.io.hypodd.pha
+    """
+
+    def setUp(self):
+        path = os.path.dirname(__file__)
+        self.fname = os.path.join(path, 'data', 'example.pha')
+
+    def test_is_pha(self):
+        self.assertEqual(pha._is_pha(self.fname), True)
```

Comment:
No test for the negative yet. Maybe just run it against all the Python files in the test folder or something like this?